### PR TITLE
Fix and stabilize some GitHub Actions workflows

### PR DESCRIPTION
**Pull request by andreydevyatkin**
Added fixes in order to stabilize some of the GitHub Actions workflows

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: .github/workflows/beam_PostCommit_Java_ValidatesRunner_Flink.yml [hunk 1]
```diff
@@ -86,6 +86,7 @@ jobs:
         uses: EnricoMi/publish-unit-test-result-action@v2
         if: always()
         with:
+          large_files: true
```
Comment (Abacn) [inline .github/workflows/beam_PostCommit_Java_ValidatesRunner_Flink.yml:89]: thanks for fixing this!

File: .github/workflows/beam_PostCommit_Java_ValidatesRunner_Direct.yml [hunk 2]
```diff
@@ -75,9 +75,7 @@ jobs:
           distribution: 'zulu'
           java-version: '8'
       - name: run validatesRunner script
-        uses: ./.github/actions/gradle-command-self-hosted-action
-        with:
-          gradle-command: :runners:direct-java:validatesRunner
+        run: ./gradlew :runners:direct-java:validatesRunner
```
Comment (Abacn) [inline .github/workflows/beam_PostCommit_Java_ValidatesRunner_Direct.yml:78]: These changes from using composite action to a command line directly. What was the problem for gradle-command-self-hosted-action here, should we focus on fix the action instead?

If it is some default options in `gradle-command-self-hosted-action`, should we have an option to control those options?

File: .github/workflows/beam_PostCommit_Python_Examples_Flink.yml [hunk 3]
```diff
@@ -53,7 +53,7 @@ jobs:
       github.event_name == 'workflow_dispatch' ||
       startsWith(github.event.comment.body, 'Run Python Examples_Flink')
     runs-on: [self-hosted, ubuntu-20.04, main]
-    timeout-minutes: 120
+    timeout-minutes: 240
```
Comment (Abacn) [inline .github/workflows/beam_PostCommit_Python_Examples_Flink.yml:56]: if it takes so long might due to some other issue, the same test on Jenkins takes short time to finish: https://ci-beam.apache.org/view/PostCommit/job/beam_PostCommit_Python_Examples_Flink/buildTimeTrend

**Review by Abacn — COMMENTED** [review]
Thanks for the fix! Left a couple of questions below

File: .github/workflows/beam_PostCommit_Java_ValidatesRunner_Direct.yml (hunk 2 above)
Comment (andreydevyatkin) [reply .github/workflows/beam_PostCommit_Java_ValidatesRunner_Direct.yml:78]: Yes, I noticed that the Jenkins job doesn't use the default arguments like `-Pdocker-pull-licenses`. As an alternative way we can specify the `default-arguments` as empty:
```
      - name: run validatesRunner script
        uses: ./.github/actions/gradle-command-self-hosted-action
        with:
          gradle-command: :runners:direct-java:validatesRunner
          default-arguments: ''
```
But there is a `max-workers` argument that should also be handled somehow. So, I found running it as a command line easier than changing action and adding additional validation.

**Review by Abacn — APPROVED** [review]

**Pull request merged**